app/models/course.rb

```diff
@@ -149,6 +149,27 @@ def application_forms
       .distinct
   end
 
+  def self.find_courses_in_previous_cycle(provider)
+    previous_cycle
+        .joins(:course_options)
+        .distinct
+        .where(provider: provider)
+        .includes(%i[provider accredited_provider])
+  end
+
+  def self.find_courses_in_current_cycle(provider)
+    current_cycle
+        .open_on_apply
+        .joins(:course_options)
+        .distinct
+        .where(provider: provider)
+        .includes(%i[provider accredited_provider])
+  end
+
+  def self.unique_ratified_courses(provider)
```

Comment:
The name implies this has something to do with the ratifying provider but the implementation is any course where the provider isn't the training provider. 
The join doesn't make sense here in this context either.

Reply:
I've removed this method, and kept the logic in the `get_courses_ratified_by_provider.rb` query class